app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala

```diff
@@ -315,6 +315,24 @@ trait ValueParser extends RegexParsers with PackratParsers with BasicParsers {
     | periodFun ^^ { case _ ~ dateExpr1 ~ _ ~ dateExpr2 ~ _ =>
       Period(DateCtx(dateExpr1), DateCtx(dateExpr2))
     }
+    | ("daysBetween(" ~ dateExpr ~ "," ~ dateExpr ~ ")" ~ ".sum|".r ^^ {
+      case _ ~ (dateExpr1: DateExpr) ~ _ ~ (dateExpr2: DateExpr) ~ _ ~ prop =>
+        prop match {
+          case ".sum" =>
+            Between(DateCtx(dateExpr1), DateCtx(dateExpr2), MeasurementType.SumDays)
+          case _ =>
+            Between(DateCtx(dateExpr1), DateCtx(dateExpr2), MeasurementType.Days)
+        }
```

Comment:
We should not handle `sum` any special. 

We already have 

```
final case class Sum(field1: Expr) extends Expr
```

for handling sums so we should use it as

```
Sum(Between(DateCtx(dateExpr1), DateCtx(dateExpr2), MeasurementType.Days))
```

Note. On the frontend we turn `Sum` into series of additions 
```
Between(DateCtx(1_dateExpr1), DateCtx(1_dateExpr2), MeasurementType.Days) +
Between(DateCtx(2_dateExpr1), DateCtx(2_dateExpr2), MeasurementType.Days) +
Between(DateCtx(3_dateExpr1), DateCtx(3_dateExpr2), MeasurementType.Days)
```